src/plugins/cloudMetadata.nim

```diff
@@ -199,15 +199,33 @@ proc cloudMetadataGetrunTimeHostInfo*(self: Plugin, objs: seq[ChalkObj]):
   #
   if isGoogleHost(vendor) and
     (isSubscribedKey("_GCP_INSTANCE_METADATA") or
+    isSubscribedKey("_GCP_PROJECT_METADATA") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_IP") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_REGION") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_TAGS") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_ACCOUNT_INFO") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_SERVICE_TYPE") or
     isSubscribedKey("_OP_CLOUD_PROVIDER_INSTANCE_TYPE")):
+    trace("Querying GCP metadata")
+    if isSubscribedKey("_GCP_PROJECT_METADATA"):
+      let projectOpt = hitProviderEndpoint("http://169.254.169.254/computeMetadata/v1/project/?recursive=true", newHttpHeaders([("Metadata-Flavor", "Google")]))
+      if projectOpt.isSome():
+        let valueProj = projectOpt.get()
+        try:
```

Comment:
why 2 try excepts for a single chalk key?in the last comment I meant to move the `parseJson` as well as `nimJsonToBox` into a single `try..except`

Reply:
there were two different errors traced but can combine